os/kernel/log_dump/log_dump.c

```diff
@@ -157,7 +157,8 @@ int log_dump_read_wake(void)
 		return ERROR;
 	}
 
-	int	msg_compress = true;
+	int msg_compress = true;
```

Comment:
In the commit title, I think that 

> The variable writesize provides the output buffer size to the uncompression algorithm.

should be changed to 

> The variable writesize provides the output buffer size to the compression algorithm.

Reply:
Done